morpheus/stages/postprocess/validation_stage.py

```diff
@@ -168,7 +169,10 @@ def _do_comparison(self, messages: typing.List[MultiMessage]):
             if (self._index_col.startswith("_index_")):
                 results_df.index.name = str(results_df.index.name).replace("_index_", "", 1)
 
+        print("Processing on thread\n\n\n\n\nSTUFF: {}".format(threading.current_thread().ident))
         val_df = self._filter_df(self._val_df)
+        # del self._val_df
+        # val_df = self._filter_df(read_file_to_df(self._val_file_name, FileTypes.Auto, df_type="pandas"))
```

Comment:
did you mean to leave this in?

Reply:
Absolutely not. Good catch. This is why I needed another pair of eyes on these changes.